.github/workflows/tests.yaml

```diff
@@ -0,0 +1,77 @@
+name: Tests
+on: [push, pull_request_target]
+
+jobs:
+  tests:
+    name: Unit Tests
+    runs-on: ubuntu-latest
+    if: >
+      github.event_name == 'push' || 
```

Comment:
Anyway to move this into a common variable or something as I saw the same condition in all the if's.

Not really sure if possible with Github actions. Only saw the option to define something at the workflow level as an environment variable.

Reply:
Do you mean the `push` and `pull` to be constant define somewhere ?
If so, I would say that this would probably have the inconvenient of not being the same as on the other repos... and also that I am not sure that this may have been generated like that by Github itself when you use the UI to create a workflow